src/llvm-version.h

```diff
@@ -15,3 +15,5 @@
 #if JL_LLVM_VERSION < 100000
 #define Align(a) (a)
 #endif
+
+#define LLVM_DISABLE_ABI_BREAKING_CHECKS_ENFORCING 0
```

Comment:
Why is this necessary now?

Reply:
I'm tired of carrying the patch to upstream

Reply:
Which LLVM patch is that? I thought these checks were useful to avoid accidentally linking the wrong LLVM ABI.

Reply:
careful--I'm disabling not disabling the checks here. https://github.com/JuliaLang/julia/blob/6aa0d924ff5431318a6c79c7c9ce3a87297170d0/deps/patches/llvm-6.0-DISABLE_ABI_CHECKS.patch

Reply:
> careful--I'm disabling not disabling the checks here.

You are if the patch is applied though, right?

Reply:
That patch is morally wrong and apparently I used to not understand C pre processor flags. I am happy with dropping that patch.